### DISCUSSION TITLE
Compute half way warp  for both affine and nonlinear

### DISCUSSION THREAD

**Discussion by MRIDude** [ANTsX/ANTs#1517] [Q&A]
How does one compute a halfway warp  for both an affine and nonlinear warp.

**Comment by ntustison** [comment]
If you're asking if that is something that is done in ANTs registration, then that's not something that is generated or considered during optimization.

**Comment by MRIDude** [comment]
Sorry, should have been more specific.   What I meant was when I already have a transform generated by ANTS reg, how do I compute the halfway warp from those transforms.

**Comment by ntustison** [comment]
We don't have anything in the toolkit that'll do that.
